src/gui/issueswidget.cpp

```diff
@@ -281,6 +295,16 @@ void IssuesWidget::slotItemCompleted(const QString &folder, const SyncFileItemPt
     _model->addProtocolItem(ProtocolItem { folder, item });
 }
 
+void IssuesWidget::filterDidChange()
+{
+    int filterCount = _statusSortModel->filterCount();
```

Comment:
I can't really follow what why count here and why we increase it if the regexp is empty.
Could you add a comment?